api/spec/src/paths/tags/single/getProjects/get.yaml

```diff
@@ -0,0 +1,18 @@
+description: 'Get all projects which include following tag'
```

Comment:
Two notes here:  
- it should accept array of `tagId` so api could return project for more complex queries
- we are getting projects, so it should be under project directory, and the get path should look something like `api/projects/tag/<array of ids in GET format>`